### ISSUE TITLE
ios - [[GIDSignIn sharedInstance] signOut] not working

### ISSUE THREAD

**Issue by KJow** [googlesamples/google-services#326]
I am implement the google sign out function for my ios app (objective c),

-(IBAction)GoogleSignOut:(id)sender{
    [[GIDSignIn sharedInstance] signOut];
    [[GIDSignIn sharedInstance] disconnect];
}
But I have no luck, when I sign out and sign in again, I still can sign in without password.
And I had checked the value of  [GIDSignIn sharedInstance].hasAuthInKeychain, when I sign out, it will return false, but I'm not currently sign out.

**Comment by s9027059** [comment]
Have the same issue here with SDK v4.0.2
User sign in by tap GIDSignInButton, the [GIDSignIn signOut] used to work fine until last week, this caused user can't switch account within our App.

note: had tried SDK v4.1.2 but no luck.

**Comment by denggoon** [comment]
Yeah same issue here - users cannot switch account within our app too. What is happening?

**Comment by s9027059** [comment]
Login process should redirect to a page with **existed login account** (if any) or **Use another account** options after tapped GIDSignInButt, but now that page is gone once you have signed in succeed.
There are 2 workarounds can bring that page back for now:

1. Clear Website Data of Safari in settings
2. Install another App need Google account  (e.g. Gmail, Google drive...)

**Comment by zanesc** [comment]
I am also experiencing the same issue.

**Comment by alokfuterox** [comment]
I am also experiencing the same issue.

**Comment by babac87** [comment]
Same issue. Tried with workaround with SFSafariViewController calling @"https://accounts.google.com/Logout", but it doesn't work either.

**Comment by dreamBegin** [comment]
Any updates on this issue? i'm stuck too

**Comment by jooohhn** [comment]
Still experiencing this issue, hope there'll be an update

**Comment by algrid** [comment]
I have the same problem. When I try to login again after a signout no account selection screen is presented and no password is required.

**Comment by acenspades** [comment]
Any update on this issue?
I'm also using [[GIDSignIn sharedInstance] signOut] to try and log a user out.

**Comment by algrid** [comment]
Looks like it was fixed in GoogleSignIn 5.0

**Comment by acenspades** [comment]
> Looks like it was fixed in GoogleSignIn 5.0

Thanks for the prompt reply.  I installed the Google SDK using Cocaopods.  I am assuming that when the Google SDK was installed that it was the latest version, correct?  Instead of assuming, how can i find out which version of the Google SDK is installed on my workspace project?

thanks again

**Comment by algrid** [comment]
@acenspades you can find versions of your pods in Podfile.lock
